net/stubby/files/stubby.yml

```diff
@@ -19,6 +19,8 @@ listen_addresses:
   - 127.0.0.1@5453
   -  0::1@5453
 
+appdata_dir: "/tmp/stubby"
```

Comment:
Following Debian convention, semi-persistent working data like trust anchors should be in `/var/lib/stubby`

Reply:
Yep, makes sense, thanks.